frontend/src/features/apartment/components/ApartmentMaximumPriceBreakdownModal.tsx

```diff
@@ -137,10 +137,6 @@ const MarketPricePre2011Breakdown = ({calculation}: {calculation: IIndexCalculat
                 label="+ Indeksin muutos"
                 value={calculation.calculation_variables.index_adjustment}
             />
-            <BreakdownValue
-                label="+ Huoneistokohtaiset parannukset"
-                value={calculation.calculation_variables.apartment_improvements.summary.accepted_value}
-            />
```

Comment:
I believe the ticket comment was outdated.
In Pre-2011 apartments, the "Huoneistokohtaiset parannukset" should still be listed (they are used in the calculations), but in after-2011 apartments they will never be any so they should not be in the breakdown (and they already are not) (refs. last page of [this](https://www.hel.fi/static/kanslia/Julkaisut/2021/Hitas/Liite_Hitas-j%C3%A4rjestelm%C3%A4n_tavoitteiden_toteutumista_koskeva_selvitys.pdf))